x/mongo/driver/integration/scram_test.go

```diff
@@ -133,15 +135,15 @@ func testScramUserAuthWithMech(t *testing.T, c scramTestCase, mech string) error
 func runScramAuthTest(t *testing.T, cs connstring.ConnString) error {
 	t.Helper()
 	topology := testutil.TopologyWithConnString(t, cs)
-	ss, err := topology.SelectServerLegacy(context.Background(), description.WriteSelector())
+	ss, err := topology.SelectServer(context.Background(), description.WriteSelector())
```

Comment:
nit: It seems like this variable was named `ss` because `SelectServerLegacy` returned a `SelectedServer`. Perhaps rename to `server`?

Reply:
Yep, sounds good!